apps/src/templates/HourOfCodeGuideEmailDialog.jsx

```diff
@@ -144,6 +141,7 @@ function HourOfCodeGuideEmailDialog({isSignedIn, unitId}) {
               text={isSendInProgress ? i18n.inProgress() : emailGuideButtonText}
               onClick={validateAndSave}
               color={Button.ButtonColor.brandSecondaryDefault}
+              disabled={isSendInProgress}
```

Comment:
@bethanyaconnor - added this here (I think it was your comment in the bug bask doc)

Reply:
Thank you!